library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -581,6 +653,7 @@ protected void replay(long windowId)
                 closeFile(inputStream);
               }
               processedFiles.add(recoveryEntry.file);
+
```

Comment:
Leaving this in since it helps to visually separate the following block of code.